compiler/damlc/daml-prim-src/LibraryModules.daml

```diff
@@ -5,7 +5,7 @@
 {-# OPTIONS_GHC -fno-warn-unused-imports #-}
 
 
--- Remove this module when DEL-6311 is done.
+-- | HIDE Remove this module when DEL-6311 is done.
```

Comment:
Gotta love comments that refer to jira tickets that have been fixed.

Reply:
Can't we just remove this file?

Reply:
Thinking about it, we fixed this for `damlc build` but we use `damlc compile` to compile daml-stdlib so it doesn’t help.

Reply:
Probably ... I'm going to go through the stdlib in a separate PR to clean up docs and I'll try removing it then.